assets/stylesheets/kitten/tokens/_borders.scss

```
@@ -24,5 +24,6 @@
   --border-radius-s: 6px; /* 6px */
   --border-radius-m: 8px; /* 8px */
   --border-radius-l: 10px; /* 10px */
-  --border-radius-xl: 100px; /* 100px */
+  --border-radius-xl: 20px; /* 20px */
+  --border-radius-xxl: 250px; /* 250px */
```

Pour être plus explicite (et laisser de la place pour des border-radius plus gros), qu'est-ce que tu penses de remplacer le nom par `-rounded` ? De fait, cette valeur est hors de l'échelle `s` à `xxl`

```suggestion
  --border-radius-rounded: 250px; /* 250px */
```